mlflow/tracking/fluent.py

```diff
@@ -988,6 +989,100 @@ def log_image(image: Union["numpy.ndarray", "PIL.Image.Image"], artifact_file: s
     MlflowClient().log_image(run_id, image, artifact_file)
 
 
+def log_table(
```

Comment:
Curious how we should use `log_table` in `mlflow.evaluate`. Can we just call it with `artifact_file=LLM_ARTIFACT_NAME` like below?

```python
# mlflow.evaluate

def evaluate(...):
    ...
    mlflow.log_table(data, LLM_ARTIFACT_NAME)
```

where `LLM_ARTIFACT_NAME` represents a special file name.